src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -37,10 +37,10 @@ datasets:
     years: [2019, 2020]
   eia:
     eia923:
-      years: [2021, 2022]
+      years: [2020, 2021, 2022]
     eia860:
-      years: [2021, 2022]
-      eia860m: false # true; change back when add eia860m
+      years: [2020, 2021, 2022]
```

Comment:
This is a good question. I don't really know the answer tbh. I think the idea is that we don't need to add more training data because it's supposed to train the model to fill it in. But we could certainly add a few rows of training data for a new year if we needed to.It was a a big undertaking to do all the top utilities.